.github/workflows/push.yml

```
@@ -298,92 +320,92 @@ jobs:
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-
       - name: Prepare Docker build arguments
         id: dockerargs
         if: steps.gather.outcome == 'success'
         env:
           NODE_IMAGE_VERSION: ${{ needs.prepare.outputs.NODE_IMAGE_VERSION }}
           GIT_COMMIT_SHA: ${{ github.sha }}
-          DOCKER_BASE_IMAGE_REGISTRY: ${{ env.DOCKER_BASE_IMAGE_REGISTRY }}
         run: |
           set -x
+          # Create a temporary file with the NX_CLOUD_ACCESS_TOKEN
```

actionlint (1.7.4)

property "cache_image" is not defined in object type {build_chunks: string; cache_key: string; deploy_feature: string; docker_tag: string; images: string; last_good_build_docker_tag: string; node-modules-hash: string; node_image_version: string; test_chunks: string}

(expression)